docs/rules/no-import-test-files.md

```diff
@@ -0,0 +1,60 @@
+# Ensure no tests are imported anywhere
+
+This rule will verify that you don't import any test files. It will consider the root of the project to be the closest folder containing a `package.json` file, and will not do anything if it can't find one. Test files in `node_modules` will not be linted as they are ignored by ESLint.
+
```

Comment:
The rule should mention why it's bad to import test files, like described in https://github.com/avajs/eslint-plugin-ava/issues/141